app/templates/components/patient-viewer.hbs

```diff
@@ -46,7 +46,53 @@
       </div>
       <div class="panel-body">
         <div class="collapse in" id="huddleList">
+          <form class="form-horizontal form-group-striped">
+            {{#if futureDisplayHuddle}}
+              <div class="form-group">
+                <div class="next-huddle-date">
+                  <i class="fa fa-calendar-o"></i>
+                  {{moment-format futureDisplayHuddle.date 'MMM D, YYYY'}}
+
+                  <i class="fa fa-check-square-o pull-right cursor-pointer" onclick={{action attrs.openReviewPatientModal futureDisplayHuddle}}></i>
+                </div>
+              </div>
+            {{/if}}
+            <div class="add-new-filter-lg">
+              <button type="button" class="btn btn-link" data-toggle="collapse" data-target="#patientViewerPikaday" aria-expanded="false" aria-controls="patientViewerPikaday" onclick={{action 'toggleScheduleChevron'}}>
+                <i class="fa fa-calendar-o"></i> Schedule <i class="fa {{if schedulePanelOpen 'fa-chevron-down' 'fa-chevron-right'}}"></i>
+              </button>
+
+              <div id="patientViewerPikaday" class="collapse">
+                <div id="patientViewerPikadayCalendar"></div>
+                <div class="patient-viewer-schedule-block">
+                  {{moment-format selectedScheduleDate 'MMM D, YYYY'}}
 
+                  {{#if selectedScheduleHuddle}}
+                    {{!-- <i class="fa fa-edit pull-right"></i> --}}
+                    {{#unless selectedScheduleHuddlePatient.reviewed}}
+                      <i class="fa fa-check-square-o" onclick={{action attrs.openReviewPatientModal selectedScheduleHuddle}}></i>
+                    {{/unless}}
+                  {{else}}
+                    <i class="fa fa-plus-circle" onclick={{action attrs.openAddHuddleModal selectedScheduleDate}}></i>
+                  {{/if}}
+
+                  <div class="meta">
+                    {{#if selectedScheduleHuddle}}
+                      Geriatrics Huddle<br>
+                      Leader: {{selectedScheduleHuddle.displayLeader}}<br>
+                      {{selectedScheduleHuddlePatient.reasonText}}
+                      {{#if selectedScheduleHuddlePatient.reviewed}}
+                        <br>
+                        Reviewed on {{moment-format selectedScheduleHuddlePatient.reviewed 'MMM D, YYYY'}}
```

Comment:
While it's usually a good rule-of-thumb to keep the date display UI the same, this won't always be the case. We can always revisit this once we have the entire app fleshed out to see where we can optimize with helpers.